index.html

```diff
@@ -171,6 +174,9 @@ <h2>Contact Me</h2>
 			</div>
 		</section>
 		</footer>
```

Comment:
- [ ] [REQUIRED] For the contact section, kindly ensure you stick properly to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) provided. You need to make the `text-area` bigger.  Please ensure that after the implementation, your entire mobile section looks like the figma template.
### Your Implementation
![Screenshot from 2022-03-25 19-55-50](https://user-images.githubusercontent.com/58692590/160189574-dfe77385-e74a-4c36-9eb2-c4f8d1ff4c99.png)

### Figmam Template
![Screenshot from 2022-03-25 19-55-33](https://user-images.githubusercontent.com/58692590/160189603-19b50cc9-4e54-4be3-ab82-e16717561d6a.png)